pulsar/web/framework.py

```diff
@@ -183,6 +183,6 @@ def __iter__(self):
 
     def __next__(self):
         buffer = self.input.read(1024)
-        if(buffer == b""):
+        if (buffer == b""):
```

Comment:
```suggestion
        if buffer == b"":
```
so you've started writing JS too? 😆

Reply:
Don't accuse me of such a thing, John wrote it, I just added the space (but didn't think to remove the parens) :joy: